rmf_site_editor/src/widgets/inspector/inspect_model_description/inspect_circle_collision.rs

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2025 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+use super::inspect_robot_properties::RobotPropertyData;
+use crate::{
+    interaction::Selection,
+    site::{
+        Affiliation, CircleCollision, Collision, Group, ModelMarker, ModelProperty, Pose, Robot,
+        RobotProperty, RobotPropertyKind,
+    },
+};
+use bevy::prelude::*;
+use bevy_egui::egui::{DragValue, Grid, Ui};
+
+#[derive(Default)]
+pub struct InspectCircleCollisionPlugin {}
+
+impl Plugin for InspectCircleCollisionPlugin {
+    fn build(&self, app: &mut App) {
+        app.world
+            .resource_mut::<RobotPropertyData>()
+            .0
+            .get_mut(&Collision::label())
```

Comment:
You could use

```rust
.entry(Collision::label())
.or_default()
```

here so that this doesn't fail if the entry isn't already present.